docs/introduction/quick-start.md

```diff
@@ -0,0 +1,300 @@
+# Quick Start
+
+This guide will help you quickly get started with **UniCore**. You will:
+
+1. Deploy a UniCore instance locally using Docker Compose.
+2. Issue a Credential to a user.
+3. Verify that Credential.
+
+### Prerequisites
+* [Docker](https://docs.docker.com/engine/install/)
+* **UniMe** app or any other Identity Wallet that supports the following standards:
+  * **OpenID for Verifiable Credential Issuance** Working Group Draft 13
+  * **OpenID for Verifiable Presentations** Working Group Draft 20
+  * **Self-Issued OpenID Provider v2** Working Group Draft 13
+
+---
+
+## Download UniMe
+
+Download the UniMe app on your mobile device:
+
+* [App Store](https://testflight.apple.com/join/cNfBowKB)
+* [Google Play Store](https://play.google.com/store/apps/details?id=com.impierce.identity_wallet&pli=1)
+
+---
+
+## Configuration
+### 1. Clone the UniCore Repository
+Open a terminal and clone the UniCore repository:
+```bash
+git clone https://github.com/impierce/ssi-agent.git && cd ssi-agent
+```
+
+### 2. Copy the Example Configuration File
+Copying the example configuration file sets up default settings required by UniCore.
+```bash
+cp agent_application/example.config.yaml agent_application/config.yaml
+```
+
+### 3. Navigate to the Docker Directory
+```bash
+cd agent_application/docker
+```
+
+### 4. Copy the example environment file
+```bash
+cp .env.example .env
+```
+
+### 5. Set the `UNICORE__URL` Environment Variable and Update the .env File
+First, set the `UNICORE__URL` environment variable using your private IP address:
+```bash
+export UNICORE__URL=http://<your-private-ip-address>:3033
```

Comment:
Is this step even necessary? Couldn't you just set the `UNICORE__URL=http://<your-private...` directly in the `.env`?

Reply:
Added a note with extra clarification :+1: